src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -60,8 +60,57 @@ export const reviewChangeRequest = async (req: Request, res: Response) => {
     }
   });
 
+  // verify wbs element exists
+  const wbsElement = await prisma.wBS_Element.findUnique({
+    where: {
+      wbsElementId: updated.wbsElementId
+    },
+    include: {
+      workPackage: true
+    }
+  });
+
+  if (wbsElement === null)
+    return res
+      .status(404)
+      .json({ message: `wbs element with id #${updated.wbsElementId} not found` });
+
+  const progress: number | undefined = wbsElement.workPackage?.progress;
+
+  if (updated.accepted && foundCR.type === CR_Type.STAGE_GATE) {
+    prisma.work_Package.update({
+      where: { wbsElementId: wbsElement.wbsElementId },
+      data: {
+        wbsElement: {
+          update: {
+            status: WBS_Element_Status.COMPLETE,
+            changes: {
+              createMany: {
+                data: [
+                  {
+                    changeRequestId: crId,
+                    implementerId: reviewerId,
+                    detail: 'Edited WBS element status from ' + wbsElement.status + ' to "COMPLETE"'
```

Comment:
better way to do the string is `Edited WBS element status from "${wbsElement.status}" to "COMPLETE"`